package.json

```diff
@@ -45,7 +45,8 @@
     "mocha-test-container-support": "^0.2.0",
     "puppeteer": "^1.19.0",
     "stringify": "^5.2.0",
-    "watchify": "^3.11.1"
+    "watchify": "^3.11.1",
+    "didi": "latest"
```

Comment:
Can we use a concrete version number like for the other dependencies?
And is this dependency not provided by `bpmn-js` anymore like previously?

Reply:
Was left in there by mistake